### ISSUE TITLE
very small frame view in Safari 

### ISSUE THREAD

**Issue by launtony** [centreon/centreon#8692]
# BUG REPORT INFORMATION

### Prerequisites

***Versions***

```
centreon-broker-storage-20.04.0-5.el7.centos.x86_64
centreon-engine-20.04.0-3.el7.centos.x86_64
centreon-widget-graph-monitoring-20.04.0-5.el7.centos.noarch
centreon-connector-ssh-20.04.0-2.el7.centos.x86_64
centreon-base-config-centreon-engine-20.04.0-7.el7.centos.noarch
centreon-plugin-Applications-Haproxy-Snmp-20200410-101650.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Poller-20200410-101650.el7.centos.noarch
centreon-clib-20.04.0-7.el7.centos.x86_64
centreon-widget-servicegroup-monitoring-20.04.0-5.el7.centos.noarch
centreon-widget-global-health-20.04.0-5.el7.centos.noarch
centreon-license-manager-20.04.1-1.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Map4-Jmx-20200410-101650.el7.centos.noarch
centreon-plugin-Hardware-Printers-Generic-Snmp-20200410-101650.el7.centos.noarch
centreon-common-20.04.0-7.el7.centos.noarch
centreon-engine-daemon-20.04.0-3.el7.centos.x86_64
centreon-broker-cbd-20.04.0-5.el7.centos.x86_64
centreon-widget-live-top10-memory-usage-20.04.0-5.el7.centos.noarch
centreon-widget-tactical-overview-20.04.0-5.el7.centos.noarch
centreon-widget-engine-status-20.04.0-5.el7.centos.noarch
centreon-trap-20.04.0-7.el7.centos.noarch
centreon-license-manager-common-20.04.1-1.el7.centos.noarch
centreon-auto-discovery-server-20.04.0-8.el7.centos.noarch
centreon-plugin-Applications-Voip-3cx-Restapi-20200410-101650.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ldap-20200410-101650.el7.centos.noarch
centreon-plugin-Hardware-Ups-Standard-Rfc1628-Snmp-20200410-101650.el7.centos.noarch
centreon-plugin-Network-Cisco-Standard-Snmp-20200410-101650.el7.centos.noarch
centreon-plugin-Applications-Protocol-Http-20200410-101650.el7.centos.noarch
centreon-release-20.04-1.el7.centos.noarch
centreon-web-20.04.0-7.el7.centos.noarch
centreon-broker-core-20.04.0-5.el7.centos.x86_64
centreon-broker-cbmod-20.04.0-5.el7.centos.x86_64
centreon-widget-hostgroup-monitoring-20.04.0-5.el7.centos.noarch
centreon-widget-host-monitoring-20.04.0-5.el7.centos.noarch
centreon-widget-service-monitoring-20.04.0-5.el7.centos.noarch
centreon-connector-20.04.0-2.el7.centos.x86_64
centreon-gorgone-centreon-config-20.04.1-1.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ftp-20200410-101650.el7.centos.noarch
centreon-plugin-Operatingsystems-Windows-Snmp-20200410-101650.el7.centos.noarch
centreon-plugin-Applications-Protocol-Dns-20200410-101650.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Database-20200410-101650.el7.centos.noarch
centreon-20.04.0-7.el7.centos.noarch
centreon-perl-libs-20.04.0-7.el7.centos.noarch
centreon-widget-httploader-20.04.0-5.el7.centos.noarch
centreon-widget-live-top10-cpu-usage-20.04.0-5.el7.centos.noarch
centreon-pp-manager-20.04.1-1.el7.centos.noarch
centreon-database-20.04.0-7.el7.centos.noarch
centreon-gorgone-20.04.1-1.el7.centos.noarch
centreon-plugin-Applications-Databases-Mysql-20200410-101650.el7.centos.noarch
centreon-plugin-Operatingsystems-Linux-Snmp-20200410-101650.el7.centos.noarch
centreon-broker-20.04.0-5.el7.centos.x86_64
centreon-engine-extcommands-20.04.0-3.el7.centos.x86_64
centreon-widget-grid-map-20.04.0-5.el7.centos.noarch
centreon-connector-perl-20.04.0-2.el7.centos.x86_64
centreon-poller-centreon-engine-20.04.0-7.el7.centos.noarch
centreon-plugin-Applications-Webservers-Apache-Serverstatus-20200410-101650.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Central-20200410-101650.el7.centos.noarch
```

***Operating System***

*CentOS*

***Browser used***

- [x] Safari

Version: 13.1 (macOS Catalina)

***Additional environment details (AWS, VirtualBox, physical, etc.):***

### Description

Hi team, I have an issue with Safari and the version 20.04 of Centreon. On all frames (hosts, custom views....), I have a small height css. Look the screenshot above.

### Describe the received result

![window_1](https://user-images.githubusercontent.com/6367604/81148925-7e3f8f00-8f7d-11ea-91e4-1ecb89227304.png)

I have try to fix that with css properties and it works : 

![fix_1](https://user-images.githubusercontent.com/6367604/81148981-9adbc700-8f7d-11ea-98dc-6d3a28759cba.png)

It works fine with Chrome :-/

**Comment by lpinsivy** [comment]
Hi @launtony you right I confirmed this issue. We put your ticket in the backlog

**Comment by damianog** [comment]
I am testing Centreon 20.04.3 and I have the same issue on a Samsung Digital Signage Display.
When will be released the fix #8723 ?

**Comment by lpinsivy** [comment]
Hi @damianog this fix has been released for Centreon 20.04.3 as describe in the release note: https://docs.centreon.com/current/en/releases/centreon-core.html#20043

with browser do you used?

**Comment by damianog** [comment]
Hi @lpinsivy,

This issue is closed for my case.
Update 20.04.4 solved the problem with an embedded browser on Samsung digital signage player box.
Sorry but I do not know which browser engine they used.
